AdaptiveCards/templating/language.md

```diff
@@ -183,8 +189,8 @@ To assign a "data context" to any element add a `$data` attribute to the element
 This part is a bit of "dark magic". Feedback welcome.
```

Comment:
is the feedback section staying? It feels kind of weird to read this while going through the docs

Reply:
I agree - also we should remove the usage of "dark magic" from our public documentation :)